src/lib/storage/segment_encoding_utils.hpp

```diff
@@ -31,4 +29,6 @@ SegmentEncodingSpec get_segment_encoding_spec(const std::shared_ptr<const Abstra
  */
 VectorCompressionType parent_vector_compression_type(const CompressedVectorType compressed_vector_type);
 
+SegmentEncodingSpec auto_select_segment_encoding_spec(const DataType& type, const bool is_unique = false);
```

Comment:
`is_unique` > `segment_values_are_unique`?

Reply:
I don't care. What do you want?

Reply:
segment_values_are_unique